components/home/home.html

```diff
@@ -18,7 +18,7 @@
 
                 <div class="md-toolbar-tools"
                      ng-class="{'todo-section':key=='Todo','doing-section':key=='Doing','review-section':
-                     key=='Review','label-red':key=='Done',}" style="max-height: 50px">
+                     key=='Review','label-red':key=='Done',}" style="max-height: 50px; color: ghostwhite">
```

Comment:
@AsadAzizRajput  adding inline styling like `style="max-height: 50px; color: ghostwhite"` is bad practice
you have to make a class in **home.css**_(under home component)_ and add this styles in it and then apply this class on targated tag

Thank You
Inzamam Malik